package.json

```diff
@@ -501,6 +501,11 @@
                         "description": "Activate keybindings.",
                         "default": true,
                         "scope": "window"
+                    },
+                    "calva.hideREPLUi": {
```

Comment:
Purely a style thing here, but I vote for this to be cased as `calva.hideReplUi` :smiley:

Reply:
I'll consider it. There was some other setting that used this style, so I chose to follow that. But it does hurt the eye a bit.

Reply:
That's why I prefer `kebab-case` 😛 😂

Reply:
kebab-case-ftw